tests/tfhe_rust_bool/end_to_end/src/main.rs

```diff
@@ -0,0 +1,26 @@
+use clap::Parser;
+
+mod fn_under_test;
+
+// TODO(https://github.com/google/heir/issues/235): improve generality
+#[derive(Parser, Debug)]
+struct Args {
+    /// arguments to forward to function under test
+    #[arg(id = "input_1", index = 1, action)]
+    input1: u8,
+
+    #[arg(id = "input_2", index = 2, action)]
+    input2: u8,
+}
+
+fn main() {
+    let flags = Args::parse();
+    let (client_key, server_key) = tfhe::boolean::gen_keys();
+
+    let ct_1 = client_key.encrypt(flags.input1 == 0u8);
```

Comment:
Seems like True should be equality with 1, not 0, right? I know it's a bug in the test harness, but maybe add another test to catch this since the existing test doesn't cover it (unlucky!).

Reply:
I'll add it. It was done because the parser cannot (easily) process Boolean values